frontend/src/assets/css/style.css

```diff
@@ -268,3 +268,11 @@ body {
     object-fit: cover;
   }
 }
+/* CSS for Login Screen */
```

Comment:
maybe use fully tailwind instead of adding more code to .css file, with help of https://tailwindcss.com/docs/gradient-color-stops#ending-color 
you can also use bg-[#0000] to add your custom color.